### ISSUE TITLE
training  results

### ISSUE THREAD

**Issue by feidasheng** [Zhongdao/Towards-Realtime-MOT#28]
Hello, I would like to ask if your training results can be trained through the training data you published.thank  you！

**Comment by Zhongdao** [comment]
Of course : )
The retrieval accuracy of the embedding may vary, you can train more epochs for good performance (such as 60 epochs).

**Comment by feidasheng** [comment]
hello,I have processed all the training data, but I don't understand how you train the dataset path in the .json file.
![image](https://user-images.githubusercontent.com/44360716/66995002-ce015a80-f100-11e9-8d82-10dc0f0aaefa.png)  .train?   I have never written my own training path.   can  you tell  me why?  thanks !

**Comment by feidasheng** [comment]
![image](https://user-images.githubusercontent.com/44360716/67066402-078ba180-f1a5-11e9-86da-3281eac5fb55.png)
![image](https://user-images.githubusercontent.com/44360716/67066484-5a655900-f1a5-11e9-91a9-9bdcddc3824f.png)

hello，I have adjusted all of your training data sets, but in the training, the above datasets are not complete, showing that some of the images are not found, so I am currently removing these two data sets in training, thank you for your answer.

**Comment by Zhongdao** [comment]
@ feidasheng 
Sorry, at first the uploaded CUHK-SYSU dataset is incomplete, and now I have re-upload the complete CUHK-SYSU dataset, please check DATASET_ZOO.md, the complete dataset should be slightly over 1GB.

**Comment by feidasheng** [comment]
thanks ，but  do you think  your citypersons data sets have been all passed on? Why is my display still partially missing?   
![image](https://user-images.githubusercontent.com/44360716/67069844-ff396380-f1b0-11e9-98cd-ba9b2c067444.png)   this  is all ?    thank you  for  your answer  very much !

**Comment by feidasheng** [comment]
Hello, I would like to ask if the Caltech dataset you uploaded is divided into several parts. I will lose some data after decompression. Is there any good way to decompress it?  thank  you very  much.

**Comment by Zhongdao** [comment]
Yes I split the dataset into 8 parts. I am not sure whether it's because my upload is incomplete or your download is complete, but if you still have this problem you may  download  the dataset from  the original page of Caltech, It should be less problematic.

**Comment by feidasheng** [comment]
hello,I have compiled all the training data, but I need to test mot19 after the training is completed. in 
your code 
![image](https://user-images.githubusercontent.com/44360716/67139914-ed76bf80-f287-11e9-9da8-cdabbccc3b6e.png)
Where is the data set downloaded?  is here https://motchallenge.net/data/CVPR_2019_Tracking_Challenge/?  this  5.4GB?
![image](https://user-images.githubusercontent.com/44360716/67139933-20b94e80-f288-11e9-9e1e-f0ed4eeaf581.png),thank  you very much.

**Comment by Zhongdao** [comment]
Yes, that's it. However in our paper we do not report test result on mot19,  instead we use 

"test":
{
"caltech":"./data/caltech.val",
"citypersons":"./data/citypersons.val"
}

which is already included in the uploaded data.

**Comment by feidasheng** [comment]
hello,I would like to ask you how many days have you shared eight GPUs training plus tests?  thank

**Comment by Zhongdao** [comment]
I use 8x Titan Xp and the training takes about 1.5 days.

**Comment by zdmwang** [comment]
Hi, Did the queue block up during training? On line 39 of syncbn.py: ex_w, exs_w = ctx.master_queue.get (). A queue is blocked. How can I solve it? Can you help me? Thanks.

**Comment by Zhongdao** [comment]
Duplicate of #56
